src/ui/TopicExplorer.svelte

```diff
@@ -21,7 +45,9 @@
           {#each tableEntry.categories as category}
             <li class="ons-list__item">
               <a
-                href="data/{category.code}"
+                href="{config.baseUrl}/{slugify(topic.name).toLowerCase()}/{slugify(
```

Comment:
No need for the `baseUrl` set in config. We can just use the slug starting with `/` here and make it baseUrl agnostic.

Reply:
Done - I updated the config file and the href .